tools/depends/configure.ac

```diff
@@ -286,6 +292,15 @@ case $host in
     if test "x$use_cpu" = "xauto"; then
       use_cpu=$host_cpu
     fi
+    if [[[ "$host" =~ "arm64-" ]]] || [[[ "$host" =~ "aarch64-" ]]]; then
+      use_cmake_cpu=aarch64
+    elif [[[ "$host" =~ "arm32-" ]]] || [[[ "$host" =~ "aarch32-" ]]]; then
```

Comment:
On the compare places was add of 3 "[" needed, the bootstrap strip one away and on this compare it need 2 "[[".

Reply:
What is aarch32 and arm32?
There exist one variable which always contains the right value.

Reply:
Have added for this:
```
      # aarch32 related to https://en.wikipedia.org/wiki/ARM_architecture#AArch32 is v7-a
      # arm32 should never be there, only in the check to avoid possible wrong cmake CPU
```
**You mean to use the `$host_cpu` for compare?**
For them I'm sorry, that `$host_cpu` is created from `configure` itself from `$host` by `sed` like below I'm not seen direct.

It check above the host for `aarch64*-*-linux-gnu*|aarch*-*-linux-be*` with this `aarch*` other as 64 then also in `$host_cpu` not correct for cmake.

The `use_cmake_cpu=$(echo $host | sed -e 's/\-.*$//')` becomes changed to `use_cmake_cpu=$host_cpu`

Reply:
Have also looked now about this `arm*-*-linux-gnu*|arm*-*-linux-uclibc*|aarch64*-*-linux-gnu*|aarch*-*-linux-be*` in history.
The `aarch*-*-linux-be*` is thought for `aarch64` only, a bit confusing if also a aarch32 is used on the world.

See https://github.com/xbmc/xbmc/pull/10659/files#diff-25bb87ae2148a0a2abba0a8a4296b5bfR189